### PR TITLE
Vue nav

### PR DESCRIPTION
**Pull request by MCGallaspy**
## Summary

Supersedes https://github.com/learningequality/kolibri/pull/116. I'd like to get this merged so that we can continue working on navigation in a feature branch, to avoid lots of merge conflicts with a fast-changing master branch.

## TODO

- [ ] Have tests been written for the new code?
- [ ] Has documentation been written/updated?
- [ ] New dependencies (if any) added to requirements file
- [ ] Add an entry to CHANGELOG.rst
- [ ] Add yourself it AUTHORS.rst if you don't appear there

## Reviewer guidance

@radinamatic here's what you can do to inspect the html:
* Follow the getting started instructions in the readme.
* Before starting the server, enable the learn and management plugins to make navigable pages avaialble:
  * `kolibri plugin kolibri.plugins.learn enable`
  * `kolibri plugin kolibri.plugins.management enable`
* Start the server
* Go to `http://localhost:8000/learn` or `http://localhost:8000/management` to inspect html.

### CODE REVIEW

**Comment by MCGallaspy** [comment]
Ugh, I ran the tests and they pass locally. This is really frustrating.

File: kolibri/core/templates/kolibri/base.html [hunk 1]
```diff
@@ -1,19 +1,10 @@
 {% load i18n kolibri_tags webpack_tags js_reverse cache %}<html>
 <head>
   <title>{% block title %}{% endblock %} - {% trans "Kolibri" %}</title>
+  {% kolibri_main_navigation %}
```
Comment (rtibbles) [inline kolibri/core/templates/kolibri/base.html:4]: This is being loaded before any other code.

Comment (rtibbles) [inline kolibri/core/templates/kolibri/base.html:4]: Sorry, ignore - this is for bootstrapping the global var, I see.

Comment (MCGallaspy) [inline kolibri/core/templates/kolibri/base.html:4]: That is the point.

**Comment by radinamatic** [comment]
I was too slow-busy-elsewhere for this one... 😞 

Will review and comment where/if needed, or even present a PR myself! 😉

**Comment by MCGallaspy** [comment]
> Will review and comment where/if needed, or even present a PR myself! :wink:

Yeah, please fire up the server and take a look!

**Comment by radinamatic** [comment]
Ok, so if I wanted to add/change something here, should I work off and PR against the `navigation-feature-branch`?
@MCGallaspy

**Comment by MCGallaspy** [comment]
> Ok, so if I wanted to add/change something here, should I work off and PR against the navigation-feature-branch?

Yeah. I made that branch so that @DXCanas can also work off of it, so be sure to ping him on changes as well.

**Pull request merged**